README.md

```diff
@@ -1,4 +1,4 @@
-# Dmemo [![CircleCI](https://circleci.com/gh/teamdmemo/dmemo.svg?style=svg)](https://circleci.com/gh/teamdmemo/dmemo) [![Coverage Status](https://coveralls.io/repos/github/hogelog/dmemo/badge.svg?branch=master)](https://coveralls.io/github/hogelog/dmemo?branch=master)
+# Dmemo [![Coverage Status](https://coveralls.io/repos/github/hogelog/dmemo/badge.svg?branch=master)](https://coveralls.io/github/hogelog/dmemo?branch=master)
```

Comment:
Please replace CI badge, not remove it.
`https://github.com/teamdmemo/dmemo/actions/workflows/main.yml/badge.svg`
https://docs.github.com/ja/actions/managing-workflow-runs/adding-a-workflow-status-badge

Reply:
6f7b31b fixed